doc/expressionLanguage.md

```diff
@@ -454,83 +468,86 @@ In the following we provide examples of using JEXL to apply transformations.
+| `name + "value is " +value`                   | `"DevId629 value is 6"`                   | String              |
+| `{coordinates: [value,value], type: 'Point'}` | `{"coordinates": [6,6], "type": "Point"}` | GeoJSON `Object`    |
+| `ts|toisodate`                                | `2021-11-18T14:20:14.901Z`                | ISO 8601 `DateTime` |
 
 Support for `trim`, `length`, `substr` and `indexOf` transformations was added.
 
-| Expression                                                   | Expected outcome |
-| :----------------------------------------------------------- | :--------------- |
-| <code>" a "&vert;trim</code>                                 | `a`              |
-| <code>name&vert;length</code>                                | `8`              |
-| <code>name&vert;indexOf("e")</code>                          | `1`              |
-| <code>name&vert;substring(0,name&vert;indexOf("e")+1)</code> | `"De"`           |
+| Expression                                   | Expected outcome |
+| :------------------------------------------- | :--------------- |
+| `" a "|trim`                                 | `a`              |
+| `name|length`                                | `8`              |
+| `name|indexOf("e")`                          | `1`              |
+| `name|substring(0,name&vert;indexOf("e")+1)` | `"De"`           |
 
-The following are some expressions not supported by the legacy expression language:
+The following are some examples of **JEXL** expressions not supported by the **legacy** expression language:
 
-| Expression                                          | Expected outcome                    |
-| :-------------------------------------------------- | :---------------------------------- |
-| `value == 6? true : false`                          | `true`                              |
-| <code>value == 6 && name&vert;indexOf("e")>0</code> | `true`                              |
-| `array[1]+1`                                        | `3`                                 |
-| `object.name`                                       | `"John"`                            |
-| `{type:"Point",coordinates: [value,value]}`         | `{type:"Point",coordinates: [6,6]}` |
+| Expression                                  | Expected outcome                    | Format  |
+| :------------------------------------------ | :---------------------------------- | ------- |
+| `value == 6? true : false`                  | `true`                              | Boolean |
+| `value == 6 && name&vert;indexOf("e")>0`    | `true`                              | Integer |
+| `array[1]+1`                                | `3`                                 | Number  |
+| `object.name`                               | `"John"`                            | String  |
+| `{type:"Point",coordinates: [value,value]}` | `{type:"Point",coordinates: [6,6]}` | Object  |
```

Comment:
Problem rendering the simbol | placed into a code block, inside a table. Almost this part need to be reverted using : `<code> ... </code>` and `&vert;` for vertical bar <code>&vert;</code>

Reply:
Fixed 54b1a53 - it looks like GitHub's Markdown parser is less sophisticated than MkDocs/ReadTheDocs